core-services/jira-lifecycle-plugin/config.yaml

```diff
@@ -156,3 +156,18 @@ default:
     - 4.13.0
     target_version: 4.12.0
     validate_by_default: true
+orgs:
+  openshift-assisted:
+    default:
+      master:
+        add_external_link: true
+        allowed_security_levels:
+        - default
+        exclude_defaults: true
+        is_open: true
+        state_after_close:
+          status: NEW
+        state_after_merge:
+          status: ON_QA
```

Comment:
Usually, the bug status should be set to `MODIFIED` after merge. ART has a bot that moves a bug from `MODIFIED` to `ON_QA` once the bug fix has been included in a nightly build. I just want to make sure that skipping the `MODIFIED` status and moving the bug directly to `ON_QA` after merge is the intended behavior.